### PR TITLE
AZP/RELEASE: Include revision in tarball filenames

### PR DESCRIPTION
**Pull request by Alexey-Rivkin**
## What
Include RC version in the UCX source tarball name.

## Why ?
Unify naming convention.

### CODE REVIEW

File: buildlib/az-github-draft.yml [hunk 1]
```diff
@@ -12,6 +12,13 @@ jobs:
     pool:
       name: MLNX
       demands: ${{ parameters.demands }}
+    variables:
+      ${{ if eq(variables['Build.Reason'], 'ResourceTrigger') }}:
+        POSTFIX: ucx-${{ replace(variables['Build.SourceBranch'], 'refs/heads/', '') }}
```
Comment (dpressle) [inline buildlib/az-github-draft.yml:17]: why add ucx if you remove it in line 40 ?

File: buildlib/az-github-draft.yml [hunk 2]
```diff
@@ -28,6 +35,11 @@ jobs:
         # gcc --version
         ./contrib/configure-release --with-java=no
         ./contrib/buildrpm.sh -s -t -b
+
+        # Rename the source tarball
```
Comment (dpressle) [inline buildlib/az-github-draft.yml:39]: add example for name before and after

**Review by dpressle — CHANGES_REQUESTED** [review]

File: buildlib/az-github-draft.yml (hunk 2 above)
Comment (Alexey-Rivkin) [reply buildlib/az-github-draft.yml]: Done, thanks

File: buildlib/az-github-draft.yml (hunk 1 above)
Comment (Alexey-Rivkin) [reply buildlib/az-github-draft.yml]: Removed, thanks

**Review by yosefe — APPROVED** [review]

**Comment by Alexey-Rivkin** [comment]
squashed

**Comment by yosefe** [comment]
@Alexey-Rivkin seems there are unrelated changes in this PR, such as NEWS and AUTHORS files

**Comment by Alexey-Rivkin** [comment]
> @Alexey-Rivkin seems there are unrelated changes in this PR, such as NEWS and AUTHORS files

Squashed for the wrong base branch by mistake, thank you for noticing.

**Review by yosefe — APPROVED** [review]

**Review by dpressle — APPROVED** [review]

**Pull request merged**